tests/ui_tests/helpers.py

```diff
@@ -0,0 +1,110 @@
+import zlib
+import struct
+
+
+# Taken from https://pyokagan.name/blog/2019-10-14-png/, made into function and blacked
+def get_bytes_from_png(png_file: str) -> bytes:
```

Comment:
instead of implementing raw PNG reading (fragile), please use the Pillow library which we already have in dependencies

Reply:
this seems suitable: https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.tobytes

Reply:
Pillow-based impl should be small enough so that you don't need to create a new file for it, i think?